ouroboros-consensus-shelley-test/src/Test/ThreadNet/TxGen/Shelley.hs

```diff
@@ -170,3 +172,8 @@ mkGenEnv whetherPPUs coreNodes = Gen.GenEnv keySpace constants
             ksStakePools
           } =
             Gen.Presets.keySpace @(MockShelley h) constants
+
+    scriptSpace :: Gen.ScriptSpace (MockShelley h)
```

Comment:
Is this PR still in Draft because we're waiting for `cardano-ledger-specs` to export this so that we don't have to write it here?

Reply:
No, we were waiting for the function two lines below that one, `Gen.Presets.scriptSpace`. This is just a local binding.